### PR TITLE
quick fix

### PR DESCRIPTION
**Pull request by kd7tck**
Boolean errors

### CODE REVIEW

**Comment by kd7tck** [comment]
This final patch fixes some problems from the last one I pushed.

File: src/raylib.h [hunk 1]
```diff
@@ -261,7 +261,8 @@
 //----------------------------------------------------------------------------------
 #ifndef __cplusplus
 // Boolean type
-typedef enum { false, true } bool;
+#include <stdbool.h>
```
Comment (raysan5) [inline src/raylib.h]: Hi @kd7tck! Why do you need the stdbool.h? It works good with the typedef...

I did it that way for several reasons:

1) MSVC doesn't support this library until VS 2013 (http://stackoverflow.com/questions/8548521/trying-to-use-include-stdbool-h-in-vs-2010)
2) It's a new library from C99, C89 and C90 didn't support it. Despite raylib uses a minimal set of features of C99, I try to keep the full base-code as portable as possible.
3) It's easy to implement without the need of an external library (http://stackoverflow.com/questions/1921539/using-boolean-values-in-c). In raylib I always try to keep dependencies to the minimum.
4) I try to keep raylib.h clean of includes...

**Comment by kd7tck** [comment]
Is it possible to cut and paste the contents of stdbool.h into that area. I find that defining the value of true and false allows me to have pre-processor branches that check for the existence of bool. enumerating them prevents the pre-processors from checking if it exists.

File: src/raylib.h [hunk 2]
```diff
@@ -261,8 +261,9 @@
 //----------------------------------------------------------------------------------
 #ifndef __cplusplus
 // Boolean type
-#include <stdbool.h>
-//typedef enum { false, true } bool;
+    #ifndef true
+        typedef enum { false, true } bool;
+    #endif
```
Comment (raysan5) [inline src/raylib.h]: It's ok, that solution works!

**Pull request merged**